components/reference-data/src/main/java/org/datacleaner/beans/transform/RemoveDictionaryMatchesTransformer.java

```diff
@@ -50,8 +56,25 @@
 @Categorized(superCategory = ImproveSuperCategory.class, value = ReferenceDataCategory.class)
 public class RemoveDictionaryMatchesTransformer implements Transformer {
 
+    public static enum RemovedMatchesType implements HasName {
+
+        STRING, LIST;
+
+        @Override
+        public String getName() {
+            if (this == STRING) {
+                return "String";
+            } else {
+                return "List";
+            }
+
+        }
```

Comment:
Wouldn't it be better to give the names as part of the enum constructor?

```Java
STRING("String), LIST("List);

private final String _name;
private RemovedMatchesType(String name){ _name = name; }

@Override
public String getName(){ return _name;}
```

The less logic, the better. We get the added bonus of not having to change the if to if-else or a switch if other types are added.